contracts/deploy/082_frax_convex_strategy.js

```diff
@@ -108,7 +154,33 @@ module.exports = deploymentWithGovernanceProposal(
           signature: "setHarvesterAddress(address)",
           args: [cHarvester.address],
         },
-        // TODO add Harvest of FXS
+        // 4. Set OETH Oracle Router on Vault
+        {
+          contract: cVault,
+          signature: "setPriceProvider(address)",
+          args: [dOETHRouter.address],
```

Comment:
Makes me wonder if we should start using a proxy for Oracle Router contracts as well

Reply:
hmm actually thinking more about it, we would need a governor role on the proxy, which is another small addition to complexity. 

As long as the router doesn't have any complex state to maintain where a proxy contract really pays off, I'd vote not to have it as a proxy

Reply:
Good point, We can just let it be simple for now